lte/gateway/c/oai/tasks/sctp/sctp_itti_messaging.c

```diff
@@ -32,34 +32,55 @@
 
 //------------------------------------------------------------------------------
 int sctp_itti_send_lower_layer_conf(
-    task_id_t origin_task_id, sctp_assoc_id_t assoc_id, sctp_stream_id_t stream,
-    uint32_t mme_ue_s1ap_id, bool is_success) {
+    task_id_t origin_task_id,sctp_ppid_t ppid, sctp_assoc_id_t assoc_id, sctp_stream_id_t stream,
+    uint32_t ap_id, bool is_success) {
   MessageDef* msg = itti_alloc_new_message(TASK_SCTP, SCTP_DATA_CNF);
 
+  SCTP_DATA_CNF(msg).ppid	    = ppid;
   SCTP_DATA_CNF(msg).assoc_id       = assoc_id;
   SCTP_DATA_CNF(msg).stream         = stream;
-  SCTP_DATA_CNF(msg).mme_ue_s1ap_id = mme_ue_s1ap_id;
+
+  if (ppid == S1AP){
+	  SCTP_DATA_CNF(msg).mme_ue_s1ap_id = ap_id;
+  }else if(ppid == NGAP){
+	  OAILOG_INFO(LOG_SCTP, " ppid NGAP in sctp_itti_send_lower_layer_conf ");
```

Comment:
consider making this debug level. also add log to the case ppid == S1AP to make this consistent.

Reply:
log add for S1AP, like NGAP